manifests/init.pp

```diff
@@ -113,6 +113,10 @@
   # incase the jira service is managed outside of puppet. eg: using the
   # puppetlabs-corosync module: 'crm resource stop jira && sleep 15'
   $stop_jira = 'service jira stop && sleep 15',
+  # Whether to manage the 'check-java.sh' script, and where to retrieve
+  # the script from.
+  $script_check_java_manage = false,
+  $script_check_java_template = undef,
```

Comment:
If we're checking in a duplicate of the file in the tarball (see above) we can make this:
```puppet
$script_check_java_template = 'template("${module_name}/check-java.sh.erb")'
# or
$script_check_java_template = "${module_name}/check-java.sh.erb"
```